modules/core/pom.xml

```diff
@@ -402,6 +402,26 @@
             </properties>
 
             <dependencies>
+                <dependency>
+                    <groupId>org.mockito</groupId>
+                    <artifactId>mockito-core</artifactId>
```

Comment:
We already have `mockito-all` dependency which seems to cover all needs, do we need another one?

Reply:
The main reason is that `mockito-all` includes some dependencies, one of them is `harmcrest`. Unfortunately, that version conflicts with the version which is used by AI. Moreover, “mockito-all” distribution has been discontinued in Mockito 2.*.

Reply:
I've checked `>mvn dependency:tree -Dverbose` command for the `ignite-core` module.
It seems we already have `hamcrest` dependency from `junit` module.
{code}
[INFO] +- junit:junit:jar:4.11:test
[INFO] |  \- org.hamcrest:hamcrest-core:jar:1.3:test
{code}

Also,  it seems `mockito-core` is not required here. I've removed these lines from my local branch an it seems it works fine.
Am I missing something?

Reply:
Well, the problem was the version of hamcrest which is bundled with mockito-all and mockito-core (it is 1.1 if I am not mistaken),
and that version conflicted with the version included into jcache-tests (1.3 actually). So, I just excluded the former.

I just removed dependency from mockito-core and re-checked tck tests with `mvn clean test -P-release,jcache-tck -pl :ignite-core -am` and it works. So, it can be updated. Will do.